# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/style/moz.build
# Commit: ae329eaf22f7
# Full Hash: ae329eaf22f70199e5f3f9d16d4c3b7c50120732
# Author: Andrew Osmond <aosmond@mozilla.com>
# Date: 2022-06-29 03:44:30
# Regressor Bug: 1771493
# File Overlap Count: 1
# Description:
#   Bug 1771493 - Part 4. Split FontFaceSetDocumentImpl from FontFaceSetImpl. r=emilio
#   
#   This patch splits document specific functionality from FontFaceSetImpl
#   into a new class FontFaceSetDocumentImpl. This will make it easier to
#   fork FontFaceSetImpl for workers.
# ==============================================================================

diff -r f724c21b1f15 -r ae329eaf22f7 layout/style/moz.build
--- a/layout/style/moz.build	Tue Jun 28 21:47:00 2022 +0000
+++ b/layout/style/moz.build	Tue Jun 28 21:47:01 2022 +0000
@@ -142,6 +142,7 @@
     "FontFace.h",
     "FontFaceImpl.h",
     "FontFaceSet.h",
+    "FontFaceSetDocumentImpl.h",
     "FontFaceSetImpl.h",
     "FontFaceSetIterator.h",
     "MediaList.h",
@@ -192,6 +193,7 @@
     "FontFace.cpp",
     "FontFaceImpl.cpp",
     "FontFaceSet.cpp",
+    "FontFaceSetDocumentImpl.cpp",
     "FontFaceSetImpl.cpp",
     "FontFaceSetIterator.cpp",
     "FontPreloader.cpp",
